INSTALL.md

```diff
@@ -31,6 +31,10 @@ Now you can install the python dependencies by calling:
 
     $ sudo pip install -e .
 
+And the dependencies needed for development by calling:
+
+    $ sudo pip install -r requirements.txt
```

Comment:
I believe that this one should even be enough, as `requirements.txt` contains `-e .` (https://github.com/skylines-project/skylines/blob/master/requirements.txt#L3)